apps/language_server/lib/language_server/build.ex

```diff
@@ -207,6 +207,22 @@ defmodule ElixirLS.LanguageServer.Build do
     end
   end
 
+  defp purge_consolidated_protocols do
+    config = Mix.Project.config()
+    path = Mix.Project.consolidation_path(config)
+
+    with {:ok, beams} <- File.ls(path) do
+      Enum.map(beams, &(&1 |> Path.rootname(".beam") |> String.to_atom() |> purge_module()))
+    end
+
+    # Code.delete_path(path)
```

Comment:
Is there a reason not to delete the path here? It seems like it is because we don't use `Code.prepend_path`, so I guess the question is what purpose is `Code.prepend_path` serving in the phoenix code reloader. I don't feel like I personally understand why `Code.prepend_path` is needed in the Phoenix case, so I'm not sure if we would need it or not.

Reply:
I don't know that either but here `Code.delete_path(path)` doesn't seem to make any difference. BTW after compile it is prepended again.

Reply:
Okay, in that case maybe we just leave a comment explaining the reasoning.